### PR TITLE
Medinaab

### PR DESCRIPTION
**Pull request by amedinaaa**
pushing latest changes - navigation from fitness plan page to workout plan page

### CODE REVIEW

**Comment by amedinaaa** [comment]
closed an earlier PR that had conflicts (gradle related) and fixed with this PR -- hence the merge remote-tracking branch - into medinaab

File: app/src/main/java/com/example/fitnesspage/FitnessPlanApp.kt [hunk 1]
```diff
@@ -11,11 +12,20 @@ import androidx.compose.runtime.collectAsState
 import androidx.lifecycle.viewmodel.viewModelFactory
 import androidx.navigation.compose.NavHost
 import androidx.navigation.compose.composable
-import com.example.fitnesspage.ui.theme.SurveyScreen
+//import com.example.fitnesspage.ui.theme.SurveyScreen
```
Comment (madduxc) [inline app/src/main/java/com/example/fitnesspage/FitnessPlanApp.kt:15]: A minor comment about leaving commented code behind.  They get on me at work about this.

File: app/src/main/java/com/example/fitnesspage/ui/theme/FitnessPlanPage.kt [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.unit.dp
+import androidx.navigation.NavController
+
+// This composable displays information pulled from backend
+
+// Need to do:
```
Comment (madduxc) [inline app/src/main/java/com/example/fitnesspage/ui/theme/FitnessPlanPage.kt:27]: Nice use of comments to track TODO list. Asana could also help with this.

File: app/src/main/java/com/example/fitnesspage/ui/theme/FitnessPlanPage.kt [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+            }
+        }
+
+        // This shows the warning dialog if the user presses the "Back to Survey" button
+        if (showWarningDialog) {
+            AlertDialog(
+                // If clicked out of the warning pop up
+                onDismissRequest = { showWarningDialog = false },
+                title = { Text("Confirm") },
+                text = { Text("Are you sure you want to go back to the survey page? Your responses will not be saved.") },
```
Comment (madduxc) [inline app/src/main/java/com/example/fitnesspage/ui/theme/FitnessPlanPage.kt:83]: Have you tried this warning?  Does the text wrap nicely?  Just curious.

**Review by madduxc — APPROVED** [review]
Great work.  A few minor comments that will not stop the PR.

File: app/src/main/java/com/example/fitnesspage/ui/theme/FitnessPlanPage.kt (hunk 3 above)
Comment (amedinaaa) [reply app/src/main/java/com/example/fitnesspage/ui/theme/FitnessPlanPage.kt:83]: It does work correctly

File: app/src/main/java/com/example/fitnesspage/FitnessPlanApp.kt (hunk 1 above)
Comment (amedinaaa) [reply app/src/main/java/com/example/fitnesspage/FitnessPlanApp.kt:15]: duly noted, thanks.

**Pull request merged**